### ISSUE TITLE
Non-English Language Tuning on the Generation of Vocab.pkl

### ISSUE THREAD

**Issue by by-zh** [neuspell/neuspell#84]
I fine-tuned the Kazakh language and executed the following script,The script can run normally, but the automatically saved vocab.pkl file is abnormal, The content of vocab.pkl is    "{" token2idx ": {"<<PAD>>": 0,"<<UNK>>": 1,"<<EOS>>": 2}," idx2token ": {" 0 ":"<<PAD>"," 1 ":"<<UNK>>"," 2 ":"<EOS>>"}," token_freq ": ["<PAD>", - 1], ["<UNK>", - 1], ["<<EOS>>", - 1]]," pad_token ":"<<PAD>>"," unk_token ":"<<UNK>>"," eos_token ":"<EOS>>"," pad_token_idx ": 0," unk_token_idx ": 1," eos_token_idx ": 2}"

`from neuspell.commons import DEFAULT_TRAINTEST_DATA_PATH

data_dir = DEFAULT_TRAINTEST_DATA_PATH
clean_file = "sample_clean.txt"
corrupt_file = "sample_corrupt.txt"
from neuspell.seq_modeling.helpers import load_data, train_validation_split
from neuspell.seq_modeling.helpers import get_tokens
from neuspell import BertChecker

train_data = load_data(data_dir, clean_file, corrupt_file)
train_data, valid_data = train_validation_split(train_data, 0.8, seed=11690)

vocab = get_tokens([i[0] for i in train_data], keep_simple=True, min_max_freq=(1, float("inf")), topk=100000)

checker = BertChecker(device="cuda")
checker.from_huggingface(bert_pretrained_name_or_path="distilbert-base-cased", vocab=vocab)

checker.finetune(clean_file=clean_file, corrupt_file=corrupt_file, data_dir=data_dir)`

**Comment by by-zh** [comment]
Here are some Kazakh sentences that can be tested，Includes clean-data and corrupt-data
clean-data-----------------------------------------

АРМАН АРҒЫМАҚ АСАУ ОЙ ТЫНЫМ АЛМАДЫ АРМАНДЫ КЕЗІП АЛДАҒЫ
ЫСТЫҚ ЖАСЫМ ТүСТІ ЫРШЫП СУЫҚ ҚАРДЫҢ ЖүЗІНЕ АЛАҚАНҒА ТүСТІ ШЫҚ ҚАРҒА ТүСКЕН ІЗІМЕ СЕНІҢ ДЕ БАР ҚАНАТЫҢ ҚАР ҚЫЗЫ ЕДІҢ АРМАНДЫ
ДӘЛ БІР ҚИЯЛДАН ТУҒАН ЕРТЕГІ ДЕРСІҢ
ҰМЫТТЫ МА ЖОҚ ӘЛДЕ УАҚЫТТАН ТЫС ЖЕРДЕ ӨЗ ХАЯТЫН ӨЗГЕШЕ БІР КІТАПТАН ӨЗІ ОҚЫП ӨКІНІШІНДЕ УАТЫЛЫП ОТЫР МА МҰНДАҒЫ ОЛ ЖӘНЕ БІРЕУ
БІЗДІҢ ЖІГІТТЕРДІҢ БІЛІМ ДЕҢГЕЙІ ЖОҒАРЫ САБАҚ ӨТКІЗУ үРДІСІ ДЕ ОЙДАҒЫДАЙ МӘНІ МЕН ҚҰНЫ ӨТКЕННЕН ӨЗГЕШЕЛІГІ ЖАҢА ДЕҢГЕЙ ЖАҢА БИІККЕ ҰМТЫЛУЫНДА
ДӘЛ ОСЫ ТүР ТҰЛҒАСЫН КӨРІП ТҰРҒАН КЕЗДЕ МҰНЫҢ ЕСІМІН ТӘТТІ ДЕСЕҢ ЕШКІМ НАНБАСТАЙ
МЫРЗАТАЙ АҒА ТОНЫКӨКТЕЙ ТАҒЫЛЫМЫ ТЕРЕҢ ТАНЫМЫ ТҰҢҒИЫҚ ТЕГЕУРІНІ БЕРІК ТҰЛҒА ОЙШЫЛДЫҚ ПЕН МЕМЛЕКЕТШІЛДІКТІ ОТАН үШІН ЖАСАМПАЗ ҚЫЗМЕТТІ ӨЗАРА ҚАУЫШТЫРА БІЛГЕН ӨРЕЛІ ҚАЙРАТКЕР
КӨП АЙДЫҢ ЖүЗІ БОЛДЫ КӨРІСПЕГЕНІМІЗГЕ
РАХАҢНЫҢ СОҢЫНАН ЕРГЕН ЖАСТАР БІЗДІҢ АРАМЫЗДА МЫНАДАЙ БІР ӘЗІЛ БАР ЕДІ РАХЫМЖАН АҒА БІЗГЕ КӨП КӨМЕК КӨРСЕТТІ
БІРАҚ КӨБІ ГЕНЕРАЛДЫҢ ҚАНЫНДА АФРИКАЛЫҚ ГЕН БАРЫН БІЛГЕНДІКТЕН ЖАС ЖАЗУШЫҒА АРТЫҚ СҰРАҚ ҚОЮДАН АУЛАҚ ЕДІ
ТУ СЫРТЫМДАҒЫ АЙНАДАН СЫЛҚ СЫЛҚ КүЛГЕН КІМ
БІРАҚ МЕНДЕЙ БІРЕУ ТАБЫЛУЫ ХАҚ ТАЛАЙ ӘСКЕР ШАУЫП КЕТТІ
МЕН ҚАЗАҚТЫҢ АҢЫЗ ҰЛЫ БОЛУЫМА НЕГІЗ БАР
БүГІН ТүРКІСТАН САЛТАНАТ САРАЙЫНДА ДӘУІР ДАСТАНГЕРІ АТТЫ ШЫҒАРМАШЫЛЫҚ КЕШІҢІЗ ӨТПЕК
СОДАН БІР КүН БҰРЫН БІР КүНШІЛІК ЖЕРДЕГІ ӘЖЕМНІҢ БІР БАУЫРЫ ҚАТТЫ СЫРҚАТТАНЫП ЖАТЫР ДЕГЕН ХАБАР КЕЛІП ЖОЛҒА СУЫТ ШЫҚҚАНБЫЗ
ЖЕЛТОҚСАННЫҢ ШЫНДЫҒЫН ШЫМЫЛДЫҚТАП АҚЫНМЫН ДЕП АЙТУ ДА АСТАМШЫЛЫҚ
ТАБЫНДЫ КүНДІЗ ТүНІ БАҒЫП КүЗЕТУ үШІН ЖЫЛҚЫШЫЛАР БРИГАДАСЫ үШ ТОПҚА БӨЛІНДІ
БҰЛ КОНГРЕСКЕ СОНДАЙ АҚ САНКТ ПЕТЕРБОРДАН КЕЛГЕН РЕСЕЙ ҰЛТТЫҚ КІТАПХАНАСЫНЫҢ БАС ДИРЕКТОРЫ АЛЕКСАНДР ВИСЛЫЙ РЕСЕЙЛІК ЖЕТЕКШІ КІТАПХАНАЛАРДАҒЫ ЭЛЕКТРОНДЫ РЕСУРСТАР БІРЛЕСТІГІ ЖАЙЫНДА СӨЗ ҚОЗҒАДЫ
ҚАРШАДАЙ КүНІНЕН ҚҰЛАЙ СүЙГЕН ЖАНЫНА ҚОЛЫ ЖЕТПЕГЕНІ БӘЛКІМ ӨЗ СЕЗІМІН ОҒАН ДӘЛЕЛДЕЙ АЛМАҒАНЫ үШІН МӘЗ БОЛЫП ОТЫР МА
ПЕШКЕ ТОЛТЫРЫП ОТЫН САЛЫП ОЛ ОТЫНЫ ГүРІЛДЕП ЖАНҒАН КЕЗДЕ ҚОЛДАН ІСТЕЛГЕН АҒАШ ОРЫНДЫҚҚА ОТЫРА БЕРІП ҚОЛЫНДАҒЫ ХАТЫ ЕСІНЕ ТүСІП үҢІЛІП ҚАРАҒАНЫ
ОЛ БАСТЫРЫЛУДЫҢ ОСЫ БІР ҚОС СЕБЕБІНІҢ ҚАЗАҚ СОЦИУМЫНЫҢ БОЙЫНАН ТОЛЫҒЫМЕН ТАБЫЛЫП ТҰРҒАНДЫҒЫ ҒАНА
ОСЫНДАЙ КүН ПЛЮСТАУ КУЛЬТІНІҢ БАСТЫ БІР ҚАҒИДАСЫ ЖАР БАС ПЛЮС ТАУ ӨРНЕГІ
ӘРИНЕ МҰНДА ТЕОРИЯЛЫҚ БІЛІМ ҚАЖЕТ
СӨЙТІП БАЛА КүНГІ АРМАНЫМ ОРЫНДАЛДЫ

corrupt-data----------------------------------------------------------------------

АРМАН АРҒЫМАҚ АСАУ ОЙ ТЫНЫМ АЛМАДЫ АРМАНДЫ КЕЗІП АЛДАҒЫ
ЫСТЫҚ ЖАСЫМ ТүСТІ ЫРШЫП СУЫҚ ҚАРДЫҢ ЖүЗІНЕ АЛАҚАНҒА ТүСТІ ШЫҚ ҚАРҒА ТүСКЕН ІСІМЕ СЕНІҢ ДЕ БАР ҚАНАТЫҢ ҚАР ҚЫЗ ЕДІ АРМАНДЫ
ДӘЛ БІР ҚИЯЛДАН ТУҒАН ЕРТЕГІ ДЕРСІҢ
ҰМЫТТЫ МА ЖОҚ ӘЛДЕ УАҚЫТТАН ТЫС ЖЕРДЕ ӨЗ ХАЯТЫН ӨЗГЕШЕ БІР КІТАПТАН ӨЗІ ОҚЫП ӨКІНІШІНДЕ УАТЫЛЫП ОТЫР МА МҰНДАҒЫ ОЛ ЖӘНЕ БІРЕУ
БІЗДІҢ ЖІГІТТЕРДІҢ БІЛІМ ДЕҢГЕЙІ ЖОҒАРЫ САБАҚ ӨТКІЗУ үРДІСІ ДЕ ОЙДАҒЫДАЙ МӘНІ МЕН ҚҰНЫ ӨТКЕННЕН ӨЗГЕШЕЛІГІ ЖАҢА ДЕҢГЕЙ ЖАҢА БИЛІККЕ ҰМТЫЛУЫНДА
ДӘЛ ОСЫ ТүР ТҰЛҒАСЫН КӨРІП ТҰРҒАН КЕЗДЕ МҰНЫҢ ЕСІМІН ТӘТТІ ДЕСЕҢ ЕШКІМ НАНБАСТАЙ
МЫРЗАТАЙ АҒА ТОНЫКӨКТЕЙ ТАҒЫЛЫМЫ ТЕРЕҢ ТАНЫМЫ ТҰҢҒИЫҚ ТЕГЕУРІНІ БЕРІК ТҰЛҒА ОЙШЫЛДЫҚ ПЕН МЕМЛЕКЕТШІЛДІКТІ ОТАН үШІН ЖАСАМПАЗ ҚЫЗМЕТТІ ӨЗАРА ҚАУЫШТЫРА БІЛГЕН ҰРАКЕЛІ ҚАЙРАТКЕР
КӨП АЙДЫҢ ЖүЗІ БОЛДЫ КӨРІСПЕГЕНІМІЗГЕ
РАХАҢНЫҢ СОҢЫНАН ЕРГЕН ЖАСТАР БІЗДІҢ АРАМЫЗДА МЫНАДАЙ БІР ӘЗІЛ БАР ЕДІ РАХЫМЖАН АҒА БІЗГЕ КӨП КӨМЕК КӨРСЕТТІ
БІРАҚ КӨБІ ГЕНЕРАЛДЫҢ ҚАНЫНДА АФРИКАЛЫҚ ГЕН БАРЫН БІЛГЕНДІКТЕН ЖАС ЖАЗУШЫҒА АРТЫҚ СҰРАҚ ҚОЮДАН АУЛАҚ ЕДІ
ТУ СЫРТЫМДАҒЫ АЙНАДАН СЫЛҚ СЫЛҚ КүЛГЕН КІМ
БІРАҚ МЕНДЕЙ БІРЕУ ТАБЫЛУЫ ХАҚ ТАЛАЙ ӘСКЕР ШАУЫП КЕТТМ
МЕН ҚАЗАҚТЫҢ АҢЫЗ ҰЛЫ БОЛУЫМА НЕГІЗ БАР
БүГІН ТүРКІСТАН САЛТАНАТ САРАЙЫНДА ДӘУІР ДАСТАНГЕРІ АТТЫ ШЫҒАРМАШЫЛЫҚ КЕШІҢІЗ ӨТПЕК
СОДАН БІР КүНІ БҰРЫН БІР КүНШІЛІК ЖЕРДЕГІ ӘЖЕМНІҢ БІР БАУЫРЫ ҚАТТЫ СЫРҚАТТАНЫП ЖАТЫР ДЕГЕН ХАБАР КЕЛІП ЖОЛҒА СУЫТ ШЫҚҚАНБЫЗ
ЖЕЛТОҚСАННЫҢ ШЫНДЫҒЫН ШЫМЫЛДЫҚТАП АҚЫНМЫН ДЕП АЙТУ ДА АСТАМШЫЛЫҚ
ТАБЫНДЫ КүНДІЗ ТүНІ БАҒЫП КүЗЕТУ үШІН ЖЫЛҚЫШЫЛЫ БРИГАДАСЫ үШ ТОПҚА БӨЛІНДІ
БҰЛ КОНГРЕСКЕ СОНДАЙ АҚ САНКТ ПЕТЕРБОРДАН КЕЛГЕН РЕСЕЙ ҰЛТТЫҚ КІТАПХАНАСЫНЫҢ БАС ДИРЕКТОРЫ АЛЕКСАНДР ВИСЛЫЙ РЕСЕЙЛІК ЖЕТЕКШІ КІТАПХАНАЛАРДАҒЫ ЭЛЕКТРОНДЫ РЕСУРСТАР БІРЛЕСТІГІ ЖАЙЫНДА СӨЗ ҚОЗҒАДЫ
ҚАРШАДАЙ КүНІНЕН ҚҰЛАЙ СүЙГЕН ЖАНЫНА ҚОЛЫ ЖЕТПЕГЕНІ БӘЛКІМ ӨЗ СЕЗІМІН ОҒАН ДӘЛЕЛДЕЙ АЛМАҒАНЫ үШІН МӘЗ БОЛЫП ОТЫР МА
ПЕШКЕ ТОЛТЫРЫП ОТЫН САЛЫП ОЛ ОТЫНЫ ГүРІЛДЕП ЖАНҒАН КЕЗДЕ ҚОЛДАН ІСТЕЛГЕН АҒАШ ОРЫНДЫҚҚА ОТЫРА БЕРІП ҚОЛЫНДАҒЫ ХАТЫ ЕСІНЕ ТүСІП үҢІЛІП ҚАРАҒАНЫ
ОЛ БАСТЫРЫЛУДЫҢ ОСЫ БІР ҚОС СЕБЕБІНІҢ ҚАЗАҚ СОЦИУМЫНЫҢ БОЙЫНАН ТОЛЫҒЫМЕН ТАБЫЛЫП ТҰРҒАНДЫҒЫ ҒАНА
ОСЫНДАЙ КүН ПЛЮСТАУ КУЛЬТІНІҢ БАСТЫ БІР ҚАҒИДАСЫ ЖАР БАС ПЛЮС ТАУ ӨРНЕГІ
ӘРИНЕ МҰНДА ТЕОРИЯЛЫҚ БІЛІМ ҚАЖЕТ
СӨЙТІП БАЛА КүНГІ АРМАНЫМ ОРЫНДАЛДЫ
